### PR TITLE
man: document that systemctl -ff reboot does not require PID 1 to work

### PR DESCRIPTION
**Pull request by poettering**
As suggested in

https://github.com/systemd/systemd/issues/3282#issuecomment-220264509

### CODE REVIEW

File: man/systemctl.xml [hunk 1]
```diff
@@ -481,19 +481,15 @@
...
-          remounted read-only. This is hence a drastic but relatively
-          safe option to request an immediate reboot. If
-          <option>--force</option> is specified twice for these
-          operations, they will be executed immediately without
-          terminating any processes or unmounting any file
-          systems. Warning: specifying <option>--force</option> twice
-          with any of these operations might result in data
-          loss.</para>
+          <para>When used with <command>halt</command>, <command>poweroff</command>, <command>reboot</command> or
+          <command>kexec</command>, execute the selected operation without shutting down all units. However, all
```
Comment (evverx) [inline man/systemctl.xml:485]: ~~Shouldn't we update the "kexec" section too https://github.com/systemd/systemd/pull/3310/files#diff-7a3aa382affc9b9d042a71a58abdfe7fR1652 ?~~
`kexec` doesn't support `--force --force`

**Comment by poettering** [comment]
Yeah, "kexec" indeed does not support -ff, hence the current patch should be right already...

**Comment by keszybz** [comment]
Hm, the text explicitly talks about halt, poweroff, reboot, kexec, and then says "If <option>--force</option> is specified twice...". If kexec does not support "-ff" then this text is misleading.

**Comment by keszybz** [comment]
@poettering ping, see previous comment

**Comment by poettering** [comment]
ah, yeah will, sorry, kinda forgot about this PR

**Comment by poettering** [comment]
Force pushed a new version now, that explicitly clarifies that -ff does not apply to kexec.

**Pull request merged**